### PR TITLE
Update sbt and other dependencies

### PR DESCRIPTION
**Pull request by mdedetrich**
Updates SBT and other dependencies

### CODE REVIEW

File: build.sbt [hunk 1]
```diff
@@ -14,7 +14,6 @@ organization     := "com.lightbend.paradox"
 name             := "sbt-paradox-dependencies"
 
 addSbtPlugin("com.lightbend.paradox" % "sbt-paradox"          % "0.4.3")
-addSbtPlugin("net.virtual-void"      % "sbt-dependency-graph" % "0.10.0-RC1")
```
Comment (mdedetrich) [inline build.sbt:17]: Latest version of sbt already has some of this functionality

**Comment by mdedetrich** [comment]
So it turns out that sb-dependency-graph is actually needed for the logic of this plugin. This is also in newest sbt but its package private so I will need to do some code changes

File: project/build.properties [hunk 2]
```diff
@@ -1 +1 @@
-sbt.version=1.3.2
```
Comment (ennru) [inline project/build.properties:1]: This dictates the minimum sbt version for the plugin, right?
Maybe stay on something lower than latest, e.g. 1.5.0?

Comment (mdedetrich) [reply project/build.properties:1]: So I think that the reason sbt's inbuilt graph checking tool is all private is to actually not conflict with sbt-dependency-graph so lets see if it builds without problems.

Comment (mdedetrich) [reply project/build.properties:1]: Okay so it seems to work fine even with the old plugin

**Comment by mdedetrich** [comment]
@ennru False alarm, newest sbt works with the older sbt-dependency-graph

**Comment by mdedetrich** [comment]
@ennru Is there anything additional that is needed for this PR (I think you may have forgotten about this?)

**Review by ennru — APPROVED** [review]
LGTM.

**Pull request merged**